include/aspect/initial_temperature/interface.h

```diff
@@ -205,6 +205,34 @@ namespace aspect
          * in the parameter file.
          */
         std::vector<std::string> model_names;
+
+        /**
+         * A list of names of initial temperature operator strings that have been
+         * requested in the parameter file. Either one operator is given
+         * (in which case is will be used for all models), or each name is associated
+         * with a model_name, in which case each is used to modify the temperature
+        * field with the values from the current plugin. The operators should be
+        * one of: add, subtract, minimum and maximum.
+         */
+        std::vector<std::string> model_operator_names;
+
+        /**
+         * An enumerated list of operators which match the allowed
+         * model operator names which can be given in the parameter file.
+         */
+        enum OperatorList
```

Comment:
Please rename the type to `Operator`. It could be used in other structures as lists